<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidgetVenta">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabProducto">
       <attribute name="title">
        <string>Producto</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tableViewProducto">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="labelNombre">
                 <property name="text">
                  <string>Nombre:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelCodigo">
                 <property name="text">
                  <string>Codigo:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelMarca">
                 <property name="text">
                  <string>Marca:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelProveedor">
                 <property name="text">
                  <string>Proveedor:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelPrecio">
                 <property name="text">
                  <string>Precio:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelPresentacion">
                 <property name="text">
                  <string>Presentaci√≥n:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelComentario">
                 <property name="text">
                  <string>Comentario:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLineEdit" name="lineEditNombre">
                 <property name="inputMethodHints">
                  <set>Qt::ImhPreferLowercase</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditCodigo">
                 <property name="inputMethodHints">
                  <set>Qt::ImhPreferNumbers</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditMarca">
                 <property name="inputMethodHints">
                  <set>Qt::ImhPreferLowercase</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditProveedor">
                 <property name="inputMethodHints">
                  <set>Qt::ImhPreferLowercase</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditPrecio">
                 <property name="inputMethodHints">
                  <set>Qt::ImhDigitsOnly</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditPresentacion">
                 <property name="inputMethodHints">
                  <set>Qt::ImhDigitsOnly</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditComentarios">
                 <property name="inputMethodHints">
                  <set>Qt::ImhLowercaseOnly</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonCrearProducto">
             <property name="text">
              <string>Crear Producto</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConsultar">
       <attribute name="title">
        <string>Consultas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Seleccione la tabla para consultar y/o modificar</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxTablasDisponibles">
            <item>
             <property name="text">
              <string>producto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>detalle de venta</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="tableViewConsultas"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVenta">
       <attribute name="title">
        <string>Venta</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>305</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labelSeleccioneProducto">
           <property name="text">
            <string>Seleccione un producto para agregar a la venta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxListaProductos"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>38</y>
          <width>935</width>
          <height>310</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QTableView" name="tableViewProductosParaVender"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="labelProducto">
                 <property name="text">
                  <string>Nombre</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listViewNombreDeProductos"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="Codigo">
                 <property name="text">
                  <string>Codigo</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listViewCodigos"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="Precio">
                 <property name="text">
                  <string>Precio</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listViewPrecio"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Cantidad</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox"/>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Realizar Venta</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
